docs/dictionary/function/machine.lcdoc

```diff
@@ -30,22 +31,24 @@ The <machine> <function> <return|returns> a <string>.
 -   iPhone Simulator: the device is a simulated iPhone
 -   iPad: the device is the iPad
 -   iPad Simulator: the device is a simulator iPad
-
+-   : macOS, Unix and Android returns described below
 
 Description:
 Use the <machine> function to detect what type of system your
 application is running on.
 
-On Mac OS systems, the <machine> <function> uses the Gestalt() system
-call to determine the machine type.
+On <Mac OS> systems, the <machine> <function> uses the Gestalt() system
```

Comment:
This is out of date. On mac it uses `sysctlbyname` to get the `hw.model` and the output is slightly different to that here. Mine is `MacBookPro12,1`.